lib/invoker.js

```diff
@@ -7,10 +8,10 @@ module.exports = function invoker(func) {
       code: 200,
       response: undefined,
       headers: {
-        'Content-Type': 'application/json; charset=utf8',
-        'Access-Control-Allow-Methods':
+        'content-type': 'application/json; charset=utf8',
```

Comment:
Why this change, shouldn't it be case insensitive?

Reply:
You're right - it should be, and it is here where we merge user headers with event headers https://github.com/boson-project/faas-js-runtime/pull/68/files#diff-89ae878b1a78765add577244271796f156213cdd0ab6722f641ef96926f38674R54.

That change to lowercase is kind of immaterial since we do lowercase all headers later. It was left over from some debugging I was doing when I realized that I was not doing case-insensitive comparisons. But I thought it would be fine to keep. :shrug: